test/util.test.js

```diff
@@ -57,6 +57,7 @@ describe('util', () => {
 
   test('quote()', () => {
     assert.ok(quote('string') === 'string')
+    assert.ok(quote('this that') === 'this that')
```

Comment:
The `quote('this that')` MUST quote output with `$`.

Reply:
@antonmedv I think I have come at a side effect of the real issue then. I will re-attempt one more time.